client/lib/analytics/docs/tracks.md

```diff
@@ -0,0 +1,33 @@
+Analytics: Tracks
+=================
+
+Automatticians may refer to internal documentation for more information about Tracks.
+
+## Usage
+
+### `analytics.tracks.recordEvent( eventName, eventProperties )`
+
+Record an event with optional properties:
+
+```js
+analytics.tracks.recordEvent( 'calypso_checkout_coupon_apply', {
+	'coupon_code': 'abc123'
+} );
+```
+
+## Naming Conventions
+
+Event names should be prefixed by `calypso_` to make it easy to identify when analyzing the data with our various analytics tools.
```

Comment:
Unless I'm mistaken, I think we throw out any events that are not prefixed with `calypso_`